components/01-atoms/forms/select/_yds-select.scss

```diff
@@ -0,0 +1,89 @@
+@use '../../../00-tokens/tokens';
+
+// CSS-only select styling (from https://github.com/filamentgroup/select-css)
+
+.form-item__dropdown {
+  --color-dropdown-border: var(--color-gray-200);
+  --color-dropdown-border-hover: var(--color-blue-medium);
+  --color-dropdown-background: var(--color-gray-100);
+  --color-dropdown-icon: var(--color-gray-200);
+  --color-dropdown-icon-hover: var(--color-blue-medium);
+
+  border: var(--border-thickness-1) solid var(--color-dropdown-border);
+  display: block;
+  position: relative;
+
+  @media (prefers-color-scheme: dark) {
```

Comment:
@callinmullaney I cloned in what we have in place in compound and only changed the files we need to use right now. This is an instance where I haven't changed what was imported. The selects aren't in use, yet, and the CSS isn't connected. 

I wanted to leave the groundwork for selects, radios, checkboxes, in place so that styling runs a bit more quickly when we do need them (which should be soon-ish).

Reply:
Sounds like a plan